frontend/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,70 @@
+import { cloneElement } from 'react'
+import {
+  Checkbox as ChakraCheckbox,
+  CheckboxGroup,
+  CheckboxGroupProps as ChakraCheckboxProps,
+  Flex,
+  Input,
+  StylesProvider,
+  useMultiStyleConfig,
+  useStyles,
+  VStack,
+} from '@chakra-ui/react'
+
+export interface CheckboxProps extends ChakraCheckboxProps {
+  options?: string[]
+  other: boolean
```

Comment:
Wanted to include an input component as the default component so that would require the `other` boolean indicate whether or not to activate the other's option